BundleURL.swift

```diff
@@ -0,0 +1,20 @@
+import CoreML
+
+@available(macOS 10.13, iOS 11.0, tvOS 11.0, watchOS 4.0, *)
+class var BundleURL: URL{
```

Comment:
Maybe make this a struct and have a static method called `bundleUrl(modelName: String) -> URL?` or something like that